bindata/manifests/daemon/daemonset.yaml

```diff
@@ -0,0 +1,49 @@
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ingress-node-firewall-daemon
+spec:
+  selector:
+    matchLabels:
+      app: ingress-node-firewall-daemon
+  template:
+    metadata:
+      labels:
+        app: ingress-node-firewall-daemon
+        component: network
+        type: infra
+    spec:
+      containers:
+        - command: ['/bin/sh', '-c', 'mount bpffs /sys/fs/bpf -t bpf && daemon']
+          image: '{{.Image}}'
+          name: daemon
```

Comment:
Inject the node name into the container's environment variable:
~~~
      env:
        - name: NODE_NAME
          valueFrom:
            fieldRef:
              fieldPath: spec.nodeName
~~~